### ISSUE TITLE
Help with eslint rules

### ISSUE THREAD

**Issue by jahirfiquitiva** [yannickcr/eslint-plugin-react#2649]
May someone help me with the right rule configuration for this to pass?
I'm currently getting this error:
`error  Expected indentation of 12 spaces but found 13  indent`

**ESLint compliant**
```javascript
          <a role={'button'}
            className={`navbar-burger ${open ? 'is-active' : ''}`}
            href={'#'}
            aria-label={'menu'}
            aria-expanded={false}
            onClick={openMenu}>
            <span aria-hidden={'true'}/>
            <span aria-hidden={'true'}/>
            <span aria-hidden={'true'}/>
          </a>
```

**Expected/desired look**
```javascript
          <a role={'button'}
             className={`navbar-burger ${open ? 'is-active' : ''}`}
             href={'#'}
             aria-label={'menu'}
             aria-expanded={false}
             onClick={openMenu}>
            <span aria-hidden={'true'}/>
            <span aria-hidden={'true'}/>
            <span aria-hidden={'true'}/>
          </a>
``` 

Thanks in advance 🙌

**Comment by ljharb** [comment]
If you configure https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent-props.md to "first", what happens?

**Comment by jahirfiquitiva** [comment]
@ljharb I still get the same error. This is how I set it up, just in case I did it incorrectly.
`"react/jsx-indent-props": [ 2, "first" ]`

**Comment by ljharb** [comment]
In that case, it seems like a bug in the rule. A PR with a failing test case would be most helpful :-)
